src/binary.js

```diff
@@ -23,9 +23,10 @@ const RUNTIMES = {
 }
 
 // Zip a binary function file
-const zipBinary = async function(srcPath, destPath, filename, stat) {
+const zipBinary = async function(srcPath, destPath, filename, stat, runtime) {
   const { archive, output } = startZip(destPath)
   addZipFile(archive, srcPath, filename, stat)
+  addZipContent(archive, `runtime=${runtime}`, 'netlify-toolchain')
```

Comment:
as noted in https://github.com/netlify/open-api/pull/272 i'd rather use json here